### DISCUSSION TITLE
Flows being exported but not flows present in Akvorado

### DISCUSSION THREAD

**Discussion by modonovan** [akvorado/akvorado#954] [Q&A]
Hi folks,

I'm trialing Akvorado after learning of it during a technical meeting last week. I have configured my openwrt router's softflowd netflow exporter to pass traffic to my inlet service on Akvorado (running on the default udp port 2055). 

I spotted another discussion along similar lines, but in that situation flows were not actually being received on the Akvorado host.

In my situation, I can see that there are flows received...

```
akvorado_inlet_core_flows_errors{error="input and output interfaces missing",exporter="192.168.11.3"} 194381
akvorado_inlet_core_flows_http_clients 0
akvorado_inlet_core_flows_received{exporter="192.168.11.3"} 194381
```

But as you can see, allow flows received have errors (194381 flows received, 194381 flows with errors :-()

I can see that in my softflowd configuration, the interface is set so I'm really not sure what I may be doing wrong that is preventing these flows from being visualised. Would really appreciate some guidance if possible?

```
# cat  /etc/config/softflowd
config softflowd
        option enabled        '1'
        option interface      'br-lan'
<SNIP>
```

Kind regards,
-Mick

**Comment by vincentbernat** [comment]
See https://github.com/akvorado/akvorado/discussions/225. It seems there is a way to configure softflowd to export interface information.

**Reply by modonovan** [reply]
Cheers for that I should have searched for softflowd!
